src/xunit.performance.execution/BenchmarkEventSource.cs

```diff
@@ -191,14 +201,24 @@ public unsafe void BenchmarkIterationStop(string RunId, string BenchmarkName, in
                 }
             }
 
-            if (_csvWriter != null)
+            if (s_csvWriter != null)
                 WriteCSV(BenchmarkName, iteration: Iteration);
         }
 
-        public static readonly BenchmarkEventSource Log = new BenchmarkEventSource();
+        private BenchmarkEventSource() : base(true)
```

Comment:
Interesting... I was thinking that we don't recommend that this be on in production, but I'm tempted to say maybe it should be on because we want to know if there is a failure.  Perhaps you should leave this as is.